### PR TITLE
Feature: Typography

### PR DESCRIPTION
**Pull request by EdgarAM25**
add raleway and poppins typographies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,5 +51,8 @@
   "bugs": {
     "url": "https://github.com/michelada/design_system/issues"
   },
-  "homepage": "https://github.com/michelada/design_system#readme"
+  "homepage": "https://github.com/michelada/design_system#readme",
+  "description": "<p>   <a href=\"https://michelada.github.io/design_system/site/\" target=\"_blank\">     <img src=\"https://michelada.github.io/design_system/site/assets/img/sips-bits-logo.svg\" alt=\"Sips & Bits Logo\"  height=\"200\">   </a> </p>",
```
Comment (IvanRoblesMX) [inline package.json]: Change that format; those spaces between tags are necessary?
```suggestion
  "description": "<p><a href=\"https://michelada.github.io/design_system/site/\" target=\"_blank\"><img src=\"https://michelada.github.io/design_system/site/assets/img/sips-bits-logo.svg\" alt=\"Sips & Bits Logo\"  height=\"200\"></a></p>",
```

File: site/index.html [hunk 2]
```diff
@@ -112,28 +112,29 @@
...
-        <p class='font-bold'>Noto Sand Regular 16px</p>
-        <p class='text-gray-lighter'>Noto Sand Regular 16px</p>
-        <p class='text-sm'>notes 12.8px</p>
-        <small>more notes in small</small>
+        <h1>H1 Raleway - Black 40px</h1>
+        <h2>H2 Raleway - Black 32px</h2>
+        <h3>H3 Raleway - Black 24px</h3>
+        <h4>H4 Raleway - Black 20px </h4>
+        <p>p Poppins Regular 16px</p>
+        <p class='font-bold'>p_bold Poppins Bold 16px</p>
```
Comment (IvanRoblesMX) [inline site/index.html]: Is that text correct `p_bold`?  What does it mean, or why is it there?

File: site/index.html [hunk 3]
```diff
@@ -112,28 +112,29 @@
...
+        <button class="btn">Raleway - ExtraBold - 20</button>
+        <p class='text-sm'>Notes Poppins Regular 13px</p>
```
Comment (IvanRoblesMX) [inline site/index.html]: Why are some classes writing with single quotes and others with double quotes?

May you [The Team] could configure Eslint for code styling at this point before this project grows up.

File: site/index.html (hunk 2 above)
Comment (EdgarAM25) [reply site/index.html]: p_bold is a single text to reference the style of `font-bold` class in `p` tags, we are using the new design-system as reference to make this changes.

[new design system](https://www.figma.com/file/9jEtKaJ8JhC5QcB0bqWBAo/NordenDS?node-id=0%3A1)

**Review by IvanRoblesMX — APPROVED** [review]
LGTM!!

**Pull request merged**